scranShipper/index.html

```diff
@@ -4,6 +4,7 @@
     <meta charset="UTF-8"/>
     <link rel="icon" type="image/svg+xml" href="/vite.svg"/>
     <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
+    <link href="../src/RestaurantDisplay/style.css" rel="stylesheet">
```

Comment:
You shouldn't have to do this if you import the css in the RestaurantDisplay component